workos/types/fga/resource_types.py

```diff
@@ -0,0 +1,9 @@
+from typing import Any, Dict, Optional
+
+from workos.types.workos_model import WorkOSModel
+
+
+class ResourceType(WorkOSModel):
+    type: str
+    relations: Dict[str, Any]
+    created_at: Optional[str] = None
```

Comment:
Confirming `created_at` can in fact be None?

Reply:
Confirmed. I believe this is an issue with the API that will be resolved soon.